static/js/redux/actions/timetable_actions.jsx

```diff
@@ -415,11 +419,18 @@ export const addCustomSlot = (timeStart, timeEnd, day, preview, id) => (dispatch
 };
 
 export const updateCustomSlot = (newValues, id) => (dispatch) => {
-  dispatch({
```

Comment:
can we rewrite this in terms of `removeCustomSlot()`? e.g.
```javascript
if (newValues.time_start && newValues.time_start === newValues.time_end) {
  dispatch(removeCustomSlot(id))
} else {
  // original code here
}

Reply:
Done